mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +35,66 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Large Image
+   */
+  isZoomed?: boolean;
```

Comment:
Is it duplicate?

Reply:
No, they are different, isZoomable respond to click.
isZoomed immediately enlarges the photo or you can add a button and use it with it.